src/contracts/VouchRegister.sol

```diff
@@ -3,26 +3,27 @@ pragma solidity ^0.8;
 
 contract VouchRegister {
     event Vouch(
-        address solver,
-        address bondingPool,
+        address indexed solver,
+        address indexed bondingPool,
         address cowRewardTarget,
-        address txSender
+        address indexed txSender
     );
     event InvalidateVouch(
-        address solver,
-        address bondingPool,
-        address txSender
+        address indexed solver,
+        address indexed bondingPool,
+        address indexed txSender
     );
 
     constructor() {}
 
     /// @dev Allows a bonding pool to officially vouch for a solver
-    /// Anyone can call this function, but only the events where the txSender is the creator
+    /// Anyone can call this function, but only the events where the txSender is the owner
     /// of the referenced bondingPool will be officially indexed
+    /// The owner of a bonding pool is identified by the address sending the initial funding amount to the bonding pool
```

Comment:
@fleupold can you please confirm that this restriction for solver management is okay?

To my current understanding, we will use the following two safes to fund and maintain the solvers:
Gnosis bonding pool: https://gnosis-safe.io/app/eth:0x6C642caFCbd9d8383250bb25F67aE409147f78b2/transactions/queue
and 
https://cowservices.slack.com/archives/C037UV49JLR/p1651487341183539

And then this definition would work well. Wdyt?

Reply:
@Felix. Yesterday, I told you we would take the bonding pool creators as the owners, but today, I am prefering the ones who send initial funds, as these are easier to index in dune. (getting the creator in dune is a little bit more work - as the creator is not the tx.origin -, but would work as well )

I think for our safes, it does not make a difference.

Reply:
Sure, that works. Is it expected that the deposit happens in one transaction or does it take whatever account does the first bit? In this case it might be possible to front-run the bonding pool ownership (if I deposit 1COW just before the main deposit). But I guess we can tackle that if it ever happens.

Reply:
the initial funding amount referrs to the full funding amount. Let me make that clear. Then we are save in any case